config.xml

```diff
@@ -23,8 +23,8 @@
     <access origin="*" />
     <allow-intent href="http://*/*" />
     <allow-intent href="https://*/*" />
-    <preference name="android-minSdkVersion" value="21" />
-    <preference name="android-targetSdkVersion" value="28" />
+    <preference name="android-minSdkVersion" value="22" />
+    <preference name="android-targetSdkVersion" value="29" />
```

Comment:
Update README.md :)

Also, does this need an update to `ANDROID_BUILD_TOOLS_VERSION` in tools/build/Dockerfile?

Reply:
Good catch, updated README in 83a63a2.

I was able to build the app with the existing container image. I was thinking of updating it in a separate PR since it's a bigger change.